### PR TITLE
Feature/component navigation update

### LINKED ISSUE #179: Update Documentation for Navigation Component
- [x] Update demo iframe to use markup from repo https://github.com/10up/component-navigation/blob/master/index.html
    - [x] Include JS via jsdelivr https://cdn.jsdelivr.net/npm/@10up/component-navigation@latest/dist/navigation.js
    - [x] Include CSS via jsdelivr https://cdn.jsdelivr.net/npm/@10up/component-navigation@latest/dist/navigation.css
- [x] Update HTML Templated panel to remove `<header>` wrapper and site title & logo section
- [x] Update HTML Text panel to remove `<header>` wrapper and site title & logo section
- [x] Add sub section to JS section outlining how to install from npm https://github.com/10up/component-navigation/blob/master/README.md#npm
- [x] Replace SCSS section with source from PostCSS in the component repository
    - [x] Relabel SCSS to PostCSS (only for new components)
- [x] Update JS plugin link to go to https://github.com/10up/component-navigation
- [x] Update JS section to display the npm and standalone methods for initializing the component as documented in the README https://github.com/10up/component-navigation/blob/master/README.md#javascript
- [x] Update JS API section to document the component options and callback methods
- [x] Remove Sass API section

### PR DESCRIPTION
**Pull request by colorful-tones**
Fixes #179

### CODE REVIEW

File: _component/responsive-navigation/index.md [hunk 1]
```diff
@@ -18,14 +18,24 @@ iframe_height: large
 
 {% include partials/html-tabs.md %}
 
-<h3>SCSS <span class="link"><a href="scss/component.scss" target="_blank">Download SCSS</a></span></h3>
+<h3>PostCSS <span class="link"><a href="scss/component.scss" target="_blank">Download CSS</a></span></h3>
```
Comment (garand) [inline _component/responsive-navigation/index.md]: Set this to "Download PostCSS" to match the others

**Review by garand — CHANGES_REQUESTED** [review]

File: _component/responsive-navigation/index.md (hunk 1 above)
Comment (colorful-tones) [reply _component/responsive-navigation/index.md]: Good catch! Thanks

**Review by garand — APPROVED** [review]

**Comment by garand** [comment]
@timwright12 This looks good to me, could you review it when you have a moment?

**Comment by timwright12** [comment]
Holding this one back until we get the component fixed

**Pull request merged**